modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart

```diff
@@ -578,8 +578,16 @@ class Emails {
   }
 
   static String _generateVerificationCode() {
+    var lenValidationCode = AuthConfig.current.validationCodeLength;
+
+    if (lenValidationCode <= 0) {
```

Comment:
Suggestion: Move the validation into the constructor of the AuthConfig.

This won't throw until the server tries to issue a validation code. If we move it into the AuthConfig we will throw an error directly when the server is starting which should help the developer identify the configuration issue directly.

Reply:
Agree! This is a much better approach, as now (when thrown in constructor of AuthConfig) the Exception when the server tries to start.